packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -292,6 +295,13 @@ export class ActionBar extends LitElement {
 
   private handleDefaultSlotChange(): void {
     this.updateGroups();
+    if (this.layout === "horizontal-fill") {
+      const actions = queryActions(this.el);
+      actions.forEach((a) => {
+        a.alignment = "center";
```

Comment:
@josercarcamo I think we should discuss with the team whether or not we should be setting the alignment here. `alignment` is a public property that a user may have already set on the action which we would be overriding here. I'm not sure we should be doing that.